modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+    ├── _images -> ../_images
+    ├── modules -> ../modules
+    ├── assembly-kogito-configuring.adoc
+    ├── assembly-kogito-creating-running.adoc
+    ├── assembly-kogito-decision-engine.adoc
+    ├── assembly-kogito-developing-decision-services.adoc
+    ├── assembly-kogito-developing-process-services.adoc
+    ├── assembly-kogito-using-dmn.adoc
+    ├── assembly-kogito-using-drl.adoc
+    ├── ...
+├── modules
+    ├── configuration
+        ├── con-data-index-service.adoc
+        ├── con-jobs-service.adoc
+        ├── proc-data-index-service-enable-security.adoc
+        ├── ref-data-index-service-queries.adoc
+        ├── ...
+    ├── creating-running
+        ├── con-kogito-automation.adoc
+        ├── con-kogito-modelers.adoc
+        ├── proc-kogito-app-examples-running.adoc
+        ├── ref-kogito-app-examples.adoc
+        ├── ...
+    ├── decision-services  // Contains modules related to multiple assemblies.
+        ├── con-decision-tables.adoc
+        ├── con-dmn.adoc
+        ├── con-drl.adoc
+        ├── proc-dmn-model-creating.adoc
+        ├── ref-dmn-model-example.adoc
+        ├── ...
+    ├── process-services
+        ├── ...
+    ├── ...
+├── titles-enterprise
+    ├── kogito-configuring
+        ├── _images -> ../../_images
+        ├── assemblies -> ../../assemblies
+        ├── master.adoc
+        └── master-docinfo.xml
+    ├── kogito-creating-running
+        ├── ...
+    ├── kogito-developing-decision-services  // Contains multiple assemblies.
+        ├── ...
+    ├── ...
+├── pantheon2.yml
+└── README.md
+----
+
+This repository structure contains the following key components:
+
+* `_artifacts`: Directory with document attribute files and other metadata files. The underscore in the folder name is an optional indication of a non-content folder.
+* `_images`: Directory with image files, optionally divided into sub-directories by subject matter. The underscore in the folder name is an optional indication of a non-content folder.
+* `assemblies`: Directory with assembly files. You collect and compile assemblies into documents (titles) using the `titles-enterprise` folder.
+* `modules`: Directory with module files, optionally divided into sub-directories by subject matter.
+* `titles-enterprise`: Directory with folders for each enterprise document or _title_ to be built in Pantheon. Each document folder contains a `master.adoc` file where you include one or more relevant assemblies that are built as part of the document. The `master-docinfo.xml` file contains metadata that is required by `bccutil` (and `ccutil`) when converting to DocBook format. (This metadata file might be deprecated in a future release of `bccutil` and Pantheon 2.) In repositories that are single sourced with a community project, a separate `titles-community` folder might be dedicated to community configurations and output, if applicable.
+* `pantheon2.yml`: YAML file with metadata that is required for publishing content on Pantheon 2:
```

Comment:
No need to say "Directory"/"YAML file" imo. That should be obvs to users. Start with third person singular verbs.

* `_artifacts`: Contains attributes and metadata. Optionally prefix the name with an underscore character to denote a directory that does not contain content.
* `_images`: Contains image files that you can optionally organize into sub-directories. ... and so on.

"In cases where content exists in community repositories, you can create a `titles-community` directory that holds upstream only content."

Reply:
I'm listing things that are all at the same top level of the repo, most of which directories, some files (like the YAML and README). There's really not a directory to specify since it depends on the user and not shown in the structure. I could just use an example parent dir name, but then should add the same to all the other directories listed for consistency (example/titles-enterprise). Then just noise to me. Seems unnecessary imo.

Yeah, would be fine to role with verb-first, but because some are folders and the one is a file, I wanted to start with what it is (directory or file), otherwise when I get to the YAML, it will also say "Contains ..." and could be odd, even with the parent dire there named. Not trying to split hairs, just keep it simple and clear. I think people get it, but that's just me. I'm open if you have a strong feeling otherwise.

Regarding that community rewording, I don't think that's quite accurate. The whole purpose of a single-sourced repo is to have mostly or all shared content, and then the community folder dedicated to the configs to just the community rendering, which is really all that's different. For content that's community only, that's controlled by including it in the community folder rendering files and not including in the enterprise rendering files. But again, this may vary depending on circumstances, so the description I went with is for this specific repo scenario, which would follow what I've explained here.

Reply:
YAML files can contain metadata.

Sure thing with that community rewording. I just see passive and my instinct is to re-word.

Reply:
Understood.